internal/build/job/maker.go

```diff
@@ -19,18 +19,18 @@ type Maker interface {
 }
 
 type maker struct {
-	mh     build.ModuleHelper
+	helper build.Helper
 	scheme *runtime.Scheme
 }
 
-func NewMaker(mh build.ModuleHelper, scheme *runtime.Scheme) Maker {
-	return &maker{mh: mh, scheme: scheme}
+func NewMaker(helper build.Helper, scheme *runtime.Scheme) Maker {
```

Comment:
```suggestion
func NewMaker(helper build.Helper, scheme *runtime.Scheme) *maker {
```
See https://github.com/qbarrand/oot-operator/pull/82/files#r906702457

Reply:
fixed